extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -236,6 +244,24 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     super.onCreativeRender(isVerifiedAmpCreative);
     injectActiveViewAmpAnalyticsElement(
       this, this.extensions_, this.ampAnalyticsConfig);
+    this.centerCreative_();
+  }
+
+  /**
+   * Centers the outermost creative iframe inside the ad slot.
+   * @private
+   */
+  centerCreative_() {
+    const iframe = this.element.getElementsByTagName('iframe')[0];
+    if (iframe) {
+      setStyles(iframe, {
+        top: '50%',
+        left: '50%',
+        bottom: '',
+        right: '',
+        transform: 'translate(-50%, -50%)',
```

Comment:
If it's precisely logic from Doubleclick, that seems reasonable.  I'll assume that someone else sorted out the best way to do this.  (Possibly bad assumption.)